### PR TITLE
Extract Numeric#apply_min_max to #clamp

### PR DESCRIPTION
**Pull request by bdunne**
Code extracted from extensions in ManageIQ/manageiq
https://github.com/ManageIQ/manageiq/blob/e538e47a4e53190daba4f45283c58303d2afedd6/gems/pending/util/extensions/miq-numeric.rb

### CODE REVIEW

**Comment by bdunne** [comment]
@Fryguy Please review

File: lib/more_core_extensions/core_ext/numeric/clamp.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
+module MoreCoreExtensions
+  module NumericClamp
+    #
+    # Returns the square of a Numeric.
```
Comment (Fryguy) [inline lib/more_core_extensions/core_ext/numeric/clamp.rb]: This description is wrong.

Comment (bdunne) [inline lib/more_core_extensions/core_ext/numeric/clamp.rb]: Oops, that's what copy / paste gets you ;)  I corrected it.

**Comment by Fryguy** [comment]
@bdunne Can you please also update the README?

**Pull request merged**